bigquery/pom.xml

```diff
@@ -40,32 +40,28 @@
     <bigquery.test.exclude>**/BigQueryInterpreterTest.java</bigquery.test.exclude>
 
     <!-- library versions -->
-    <bigquery.api.version>v2-rev265-1.21.0</bigquery.api.version>
+    <gcp-beta.version>0.38.0-beta</gcp-beta.version>
```

Comment:
what's the status and timeline of this into a release? I"m not sure if we should build with a "beta"?

Reply:
Seems GA release versions are available now. Will use latest stable, do testing and update pull request.
https://mvnrepository.com/artifact/com.google.cloud/google-cloud-bigquery

Reply:
the version I see is 1.29.0
https://search.maven.org/#artifactdetails%7Ccom.google.cloud%7Cgoogle-cloud-bigquery%7C1.29.0%7Cjar

Reply:
Sorry for the delayed response. 
We have changed version to 1.31.0
It's working for us. The beta version and latest GA is compatible.